plugins/node/opentelemetry-instrumentation-knex/package.json

```diff
@@ -56,8 +56,9 @@
     "typescript": "5.0.4"
   },
   "dependencies": {
+    "@opentelemetry/core": "^2.0.0",
     "@opentelemetry/instrumentation": "^0.200.0",
```

Comment:
`@opentelemetry/instrumentation@0.201.0` was just released, which includes [`semconvStabilityFromStr`](https://github.com/trentm/opentelemetry-js/blob/main/experimental/packages/opentelemetry-instrumentation/src/semconvStability.ts#L81)

This dep was updated here as well, so merging changes from `main` will update to `^0.201.0`.